Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml

```diff
@@ -2588,7 +2588,11 @@ script:
       description: Identifies the origin of the data (the system that produced the
         data.
       type: String
-  dockerimage: demisto/python_pancloud_v2:1.0.0.13088
+  - name: cdl-reset-failure-times
+    deprecated: false
+    description: Reset the failure times of retrieving the access token
```

Comment:
```suggestion
    description: Use this command in case your authentication calls fail due to internal call-limit, the command will reset the limit cache.
```